internal/provider/provider.go

```diff
@@ -99,6 +101,10 @@ func (p *pingfederateProvider) Schema(_ context.Context, _ provider.SchemaReques
 				Description: "Paths to files containing PEM-encoded certificates to be trusted as root CAs when connecting to the PingFederate server over HTTPS. If not set, the host's root CA set will be used. Default value can be set with the `PINGFEDERATE_PROVIDER_CA_CERTIFICATE_PEM_FILES` environment variable, using commas to delimit multiple PEM files if necessary.",
 				Optional:    true,
 			},
+			"x_bypass_external_validation": schema.BoolAttribute{
+				Description: "Connection test will be bypassed when set to true. Default to false.",
```

Comment:
I'd mention that setting this to true adds the corresponding header to requests sent to PF.
We could also maybe just put "header" in the name of this attribute? Like "IncludeBypassExternalValidationHeader"?